test/app-python-flask-test.js

```diff
@@ -17,18 +17,35 @@ describe('python-flask', function () {
 		optionsBluemix.backendPlatform = "PYTHON";
 		return helpers
 			.run(path.join(__dirname, GENERATOR_PATH))
-			.inTmpDir()
+			.inTmpDir(function(dir){
+				const pipfile = '[[source]]\n' +
+					'\n' +
+					'url = "https://pypi.python.org/simple"\n' +
+					'verify_ssl = true\n' +
+					'name = "pypi"\n' +
+					'\n' +
+					'\n' +
+					'[dev-packages]\n' +
+					'\n' +
+					'\n' +
+					'[packages]\n' +
+					'Flask = \'==0.11.1\'\n' +
+					'gunicorn = \'==19.7.1\'';
+
+				fs.writeFileSync(dir + '/Pipfile', pipfile);
+				console.log(dir);
```

Comment:
Does the `console.log` need to be removed?

Reply:
No that's ok for the tests, it only prints out if you're doing npm test